src/EA.Iws.Web/Areas/NotificationApplication/Controllers/CarrierController.cs

```diff
@@ -235,6 +235,12 @@ public async Task<ActionResult> AddFromAddressBook(Guid id)
             var carriers = await mediator.SendAsync(new GetUserAddressBookByType(AddressRecordType.Carrier));
             model.SetCarriers(carriers.AddressRecords);
 
+            await this.auditService.AddAuditEntry(this.mediator,
+                    id,
+                    User.GetUserId(),
+                    NotificationAuditType.Create,
```

Comment:
This is using the old `Create` rather than `Added` which I've missed! Weirdly enough the CI build passed during this PR but failed once it was merged into `develop` branch. Have created #660 to fix this as I need to create a new build for Kuldeep this afternoon ready for him first thing tomorrow.